app/assets/stylesheets/modules/sitter.scss

```diff
@@ -1,23 +1,70 @@
 @import "variables";
 @import "mixins";
 
+//SEARCH
+#search-sitter-container {
+  font-family: $sitter-font-type;
+  margin-left: auto;
+  margin-right: auto;
+
+  #list-of-sitters {
+    margin-left: 9.3em;
+  }
+
+  .sitter-profile {
+    padding-top: 3em;
+    margin: 0;
+
+    .small-6 {
+      padding: 0;
+      margin:0;
+
+      img {
+        @include image-style;
+      }
+
+      .row-name {
+        text-align: center;
+      }
+    }
+
+    #sitter-info {
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 3